src/finite_field.rs

```diff
@@ -3,6 +3,8 @@
 
 //! Finite field arithmetic over a prime field using a 32bit prime.
```

Comment:
```suggestion
//! Finite field arithmetic over a prime field using a 128 bit prime.
```

Reply:
The prime hasn't changed, so this comment is correct as-is.

Reply:
Oh of course, you are correct!